ui/src/containers/SelectProposalContainer.jsx

```diff
@@ -14,7 +14,6 @@ function SelectProposalContainer() {
     if (login.selectedProposalID === null) {
       dispatch(signOut());
       serverIO.disconnect();
-      navigate('/');
```

Comment:
Unrelated, but the `signOut()` action dispatch already causes a redirect to `/login`. I'm pretty sure the `serverIO.disconnect()` call is also not needed but I'll test this separately.

Reply:
Ok